src/image/TanWcs.cc

```diff
@@ -324,8 +324,8 @@ geom::Point2D TanWcs::undistortPixel(geom::Point2D const & pix) const {
 
     // Polynomial orders for U = f(u,v) and V = g(u,v)
     // Note these may be different in general, but not usually in practise.
-    size_t const fOrder = _sipA.rows();
-    size_t const gOrder = _sipB.rows();
+    int const fOrder = _sipA.rows();
+    int const gOrder = _sipB.rows();
```

Comment:
From glancing at the code it's not obvious why these don't follow the same pattern as elsewhere -- both Tim & I did a double take. I think at least a comment pointing out what's happening would be useful.

Actually, though, these don't go negative (they're ``const``). I think the point is that you want to get a negative when you evaluate ``fOrder - i`` a few lines below. I wonder whether a cast there would be clearer than using ``int``s in the above... but I don't think it's a big deal either way.